scanner/src/main/kotlin/scanners/fossid/FossId.kt

```diff
@@ -145,13 +145,47 @@ class FossId(
             .build()
 
         service = retrofit.create(FossIdRestService::class.java)
+
+        version = runBlocking {
+            parseVersion().orEmpty()
+        }
     }
 
     override fun filterOptionsForResult(options: ScannerOptions) =
         options.mapValues { (k, v) ->
             v.takeUnless { k in secretKeys }.orEmpty()
         }
 
+    /**
+     * Extract the version version from the login page.
+     * Example: &nbsp;&nbsp;&nbsp;cli.  3.1.16 (build 5634934d, RELEASE)
+     */
+    private suspend fun parseVersion(): String? {
+        // TODO: replace with an API call when FossID provides a function (starting at version 21.2).
```

Comment:
Is there a public ticket for this or release notes which we could reference here?

Reply:
No, no publicly available information.

Reply:
Should I remove the upcoming version estimate for the feature ?

Reply:
IMO no, it's still valuable to know, even if without reference.